README.md

```diff
@@ -57,21 +58,24 @@ npm install @asgardeo/auth-react react-router-dom --save
 
 Asgardeo React SDK exposes the `AuthProvider` component, which helps you easily integrate Asgardeo to your application.
 
-First, import the `AuthProvider` component from `@asgardeo/auth-react.`
+First, import the `AuthProvider` component from `@asgardeo/auth-react.` where you applications root component is defined.
+> **Note**
+> Typically the root component of a react app is defined in the index.* file.
 
 ```TypeScript
 import { AuthProvider } from "@asgardeo/auth-react";
 ```
 Then, wrap your root component with the `AuthProvider`.
 
+
 ```TypeScript
-import React from "react";
+<strong>import React from "react";<strong>
 import { AuthProvider } from "@asgardeo/auth-react";
 
 const config = {
      signInRedirectURL: "https://localhost:3000/sign-in",
      signOutRedirectURL: "https://localhost:3000/dashboard",
-     clientID: "client ID",
+     clientID: "<client-ID>",
```

Comment:
`baseUrl` has `<org_name>` do we need to be consistent and have `<client_id>`.

Reply:
changed to client_ID